### PR TITLE
VS Wrench construct + deconstruct modes

### PR DESCRIPTION
**Pull request by DeltaNedas**
french and russian locale stuff not implemented but its golden
- vs wrench now has modes, shift r click to toggle
- right click a multiblock in deconstruction mode to deconstruct into parts

also fixed a model, name and texture issue

### CODE REVIEW

**Comment by DeltaNedas** [comment]
pls merge or something

File: src/main/java/com/best108/atom_animation_reader/impl/BasicDagNodeRenderer.java [hunk 1]
```diff
@@ -31,8 +31,8 @@ public void render(double keyframe, int brightness) {
             transformations.get(i).transform(keyframe);
             GlStateManager.translate(-customPivot.X, -customPivot.Y, -customPivot.Z);
         }
-//		Vector customPivot = pivot.createMutibleVectorCopy();
-//		GlStateManager.translate(-customPivot.X, -customPivot.Y, -customPivot.Z);
+//        Vector customPivot = pivot.createMutibleVectorCopy();
+//        GlStateManager.translate(-customPivot.X, -customPivot.Y, -customPivot.Z);
```
Comment (Rubydesic) [inline src/main/java/com/best108/atom_animation_reader/impl/BasicDagNodeRenderer.java]: Why touch other files unnecessarily

File: src/main/java/org/valkyrienskies/addon/control/item/EnumWrenchMode.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    // Construct not Assemble because it would prob be mixed up with infuser
+    CONSTRUCT("construct"), DECONSTRUCT("deconstruct");
+
+    private String name;
+
+    EnumWrenchMode(String name) {
+        this.name = name;
+    }
+
+    public String toString() {
```
Comment (Rubydesic) [inline src/main/java/org/valkyrienskies/addon/control/item/EnumWrenchMode.java]: @Override?

**Comment by Rubydesic** [comment]
I'm gonna manually merge this

**Comment by Rubydesic** [comment]
And merged with 51125b0875daaeeb7e7f92e9a28efb99a52db055

**Comment by DeltaNedas** [comment]
i used sed s/\t/    /g on src/main/java, a few files had tabs i believe thats why

**Pull request closed without merging**